library/src/scripts/splash/Splash.tsx

```diff
@@ -34,7 +35,12 @@ export class Splash extends React.Component<IProps> {
 
         return (
             <div className={classNames(className, classes.root)}>
-                <div className={classes.outerBackground} />
+                <div
+                    className={classNames(
+                        classes.outerBackground,
+                        this.props.backgroundImage && classes.customBackground(this.props.backgroundImage),
```

Comment:
Can you please make it `outerBackgroundImage` and `customOuterBackground`. For now, we only have the outer, but the component is already set up for supporting the inner background too.